ods_ci/tests/Tests/0600__distributed_workloads/test-run-kueue-e2e-tests.robot

```diff
@@ -20,13 +20,15 @@ Run Kueue E2E test
     [Tags]  Tier1
     ...     Kueue
     ...     DistributedWorkloads
+    ...     ExcludeOnDisconnected
```

Comment:
Hi @abhijeet-dhumal why it is excluded on disconnected?
My understanding was that we added s3 bucket and ray image into the disconnected env because we wanted to run these tests.

Reply:
Hii @apodhrad , The added S3 bucket is being used in [Codeflare -sdk ](https://github.com/red-hat-data-services/ods-ci/pull/1657/files#diff-0452ad2fffb0ada7c715b40ec14bcef5c29eb1b3de12d170be2f1ddee279f601R19)and [codeflare tests](https://github.com/red-hat-data-services/ods-ci/pull/1657/files#diff-9e959149cc7864aa06f33c91690012237b00164cf54d75307968dbbdad73fb87R19) to store and download MNIST datasets from. 
The Kueue component test functionality is being tested as part of  distributed workloads codeflare tests, so  we are adding `ExcludeOnDisconnected`  tag in kueue tests for now
Here are the results of the Distributed Workloads tests in disconnected : [results](https://opendatascience-jenkins-csb-rhods.apps.ocp-c1.prod.psi.redhat.com/job/rhoai/job/2.13/job/selfmanaged/[…]ted/job/psi/job/rhoai-smoke/2/robot/)